newIDE/app/src/AssetStore/ExtensionStore/index.js

```diff
@@ -117,6 +121,10 @@ export const ExtensionStore = ({
             selectedExtensionShortHeader.name
           )}
           onInstall={() => {
+            const eventsFunctionsExtension = project.getEventsFunctionsExtension(
+              selectedExtensionShortHeader.name
+            );
```

Comment:
Two things:
- you must use hasEventsFunctionsExtension, otherwise using getEventsFunctionsExtension with a non existing name will crash/returning corrupted memory content.
- let's move this in the MainFrame. It's not the responsibility of this store to handle the tabs.